java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -906,7 +904,19 @@ private void executeMemberSelect(MemberSelectExpressionTree mse) {
       ProgramState.Pop unstackMSE = programState.unstackValue(1);
       programState = unstackMSE.state;
     }
-    SymbolicValue mseValue = constraintManager.createSymbolicValue(mse);
+
+    SymbolicValue mseValue = null;
+
+    // TODO(Johan): Should this have an exception for when the symbol is unknown?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)